htdocs/wp-content/themes/hollandskroon/src/Commands/PronamicIdealCleanupCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace HK\Commands;
+
+use \WP_CLI;
+use \WP_CLI\ExitException;
+
+class PronamicIdealCleanupCommand
+{
+	/**
+	 * @throws \JsonException
+	 * @throws ExitException
+	 */
+	public function __invoke($args, $assoc_args)
+	{
+		$postsPerPage = 100;
+		$deletionCount = 0;
+
+		do {
+			// Get fulfilled payments older than 5 days in batches of 100
+			$payments = new \WP_Query([
+				'post_type'      => 'pronamic_payment',
+				'post_status'    => 'payment_completed',
+				'posts_per_page' => $postsPerPage,
```

Comment:
3. Of je skipt 1 en 2 en doet: `'posts_per_page' => -1,`
Dan kan de while loop ook weg.